### ISSUE TITLE
Differences between releases

### ISSUE THREAD

**Issue by mariantanase** [Bearsampp/Bearsampp#22]
What are the differences between the 3 packages for 1.8.3-rc1 ?
(neard, neard-basic, neard-light)

P.S. Will the packages be available for download only on github ? or do you plan to start a new domain like bearsampp ? :)

Thanks for your effort to continue the development of this LAMP!

**Comment by N6REJ** [comment]
Hi @mariantanase as for the variants, according to neards website 
![image](https://user-images.githubusercontent.com/1850089/143623225-83b3ba4d-5a50-4ccb-aade-370e40293769.png)

We will have a website ( neard.io is actually a git "pages" repo ) 😃 
I looked at the price of a .io domain and its $40/yr so I won't be going that route.  I've started a new website https://bearsampp.github.io/ but I'm focusing on getting all the repo's converted over and working with the new name first.
Right now this is a one man endeavor but hopefully more will come on board to contribute.

There are a few "weird" things in how the development of software is done that perhaps could be better, idk yet for sure.  And several wonderful things that max used that I don't know exactly how to use yet.
Most of this is trial/error on my part as I was unable to ever get guidance from max even though I asked several times over the years.
